pkgs/development/python-modules/yamlfix/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, fetchPypi
+, pytestCheckHook
+, pytest-xdist
+, pythonOlder
+, click
+, ruyaml
+}:
+
+buildPythonPackage rec {
+  pname = "yamlfix";
+  version = "0.7.2";
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1fdea35a6865c3332a227e0b75ed4c31d0d1a442c0f770353bd23406511fece9";
+  };
```

For 0.8.0 the tests will only be available in the GitHub release tarballs. 

```suggestion
  src = fetchFromGitHub {
    owner = "lyz-code";
    repo = pname;
    rev = version;
    sha256 = "sha256-Gkq80YMeiPy7xxLauA/nloW4znMV2tfE+e24HyZgUaQ=";
  };
```

---

(amend to https://github.com/NixOS/nixpkgs/pull/146994/commits/d83369676cfa770a31d4c4930cee9c2604dcee8d)